CheapShopWeb/Views/Search/Search.cshtml

```diff
@@ -31,139 +31,79 @@
+                            </div>
+                            <div class="col-sm-12 col-md-6 border border-primary">
+                                <h2>@item.name</h2>
+                                <p>Grupė: @item.group</p>
+                            </div>
+
+                            <div class="col-sm-12 col-md-3">
+                                <h2>@item.price</h2>
+                                <a href=@item.product_link class="btn btn-primary">Link</a>
+                            </div>
+                            <div class="clearfix"></div>
                         </div>
                     </div>
                 }
             }
-
-            @*<div class="productholder">
-                    <img src="http://lorempixel.com/100/100" alt=""/>
-
-                    <div class="productinfo">
-                        <h2>Product Name</h2>
-                        <p>Shop</p>
-                        <p>Category</p>
-                    </div>
-
-                    <div class="productprice">
-                        <h2>599.99</h2>
-                        <Button>Link</Button>
-                    </div>
-                </div>*@
-
-
         </div>
     </div>
 
 
-    <style>
-        #wrapper {
-            overflow: hidden; /* will contain if #first is longer than #second */
-        }
-
-        #first {
-            float: left; /* add this */
-            border: 2px solid red;
-        }
-
-        #second {
-            height: 100%;
-            border: 2px solid green;
-            overflow: hidden; /* if you don't want #second to wrap below #first */
-        }
-        
-        #boarderdefault {
-            @*border: 2px solid red*@
-            margin-bottom: 30px;
-            padding: 10px
-        }
-
-        .full-height {
-            height: 100%;
-        }
-
-        .myDiv {
-            border: 5px outset grey;
-            @*background-color: lightblue;*@
-            text-align: center;
-        }
-
-        .productholder {
-            display: flex;
-            width: 954px;
-            @* margin-bottom: 30px;
-            background-color: aqua;*@
-            border: 1px solid green;
-        }
-
-        .productholder img {
-            display: inline-block;
-            margin: 50px;
-            width: 200px;
-            height: 200px;
-        }
-
-        .productinfo {
-            border: 1px solid green;
-            display: inline-block;
-            margin: 20px 0;
-            vertical-align: top;
-            width: 500px;
-            color: black;
-            background-color:antiquewhite;
-        }
-
-        .productinfo input {
-            float: right;
-            margin-top: 20px;
-            color: ;
-        }
-
-        .productprice {
-            width: 150px;
-            float: right;
-            margin: 20px;
-            text-align: center;
-        }
-    </style>
+   
 
 </div>
+}else{
+<div><p class="h4">Nothing</p></div>
```

Comment:
čia jeigu nebūtų produktų?

Reply:
Siuo atvėju taip, bet dabar dar biški pakeičiau kad ir tuo atveju kai search null empty arba tik is whitespace'u padarytas